src/App.vue

```diff
@@ -234,6 +234,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   @Action setSummaryMode!: ActionBindingIF
   @Action setFilingType!: ActionBindingIF
   @Action setFilingId!: ActionBindingIF
+  @Action setFeePrices!: ActionBindingIF
```

Comment:
Is this used?

Reply:
If staff payment is needed then this possibly belongs in the filing views (change, correction, etc), not here in App.vue. Check what the other filing views are doing for setting fee prices or updating staff payment data.

Reply:
You may have copied this code from Create UI. It's used for a different reason there (not staff payment).

Reply:
Well actually, I see `setFeePrices` is also used in Edit UI -- in the filing views themselves, not App.vue. If you look at that code, it will hopefully make sense what it's used for (not staff payment).

Reply:
No I will drop it.